### PR TITLE
Add check for primary keys with serial types

### PR DESCRIPTION
**Pull request by mfaulther**
Closes https://github.com/mfvanek/pg-index-health/issues/317

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ All checks can be divided into 2 groups:
...
 | 18 | B-tree indexes [on array columns](https://habr.com/ru/articles/800121/)                                                            | static             | [sql](https://github.com/mfvanek/pg-index-health-sql/blob/master/sql/btree_indexes_on_array_columns.sql)            |
 | 19 | [Sequence overflow](https://habr.com/ru/articles/800121/)                                                                          | **runtime**        | [sql](https://github.com/mfvanek/pg-index-health-sql/blob/master/sql/sequence_overflow.sql)                         |
+| 20 | Primary keys with [serial types](https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_serial)                                | static             | [sql](ttps://github.com/mfvanek/pg-index-health-sql/blob/master/sql/primary_keys_with_serial_types.sql)             |                                                                                                                     |
```
Comment (mfvanek) [inline README.md:70]: Broken link. See ` [sql](ttps:`

File: spring-boot-integration/pg-index-health-test-starter/src/main/java/io/github/mfvanek/pg/spring/DatabaseStructureHealthAutoConfiguration.java [hunk 2]
```diff
@@ -263,4 +272,5 @@ public StatisticsMaintenanceOnHost statisticsMaintenanceOnHost(final PgConnectio
     public ConfigurationMaintenanceOnHost configurationMaintenanceOnHost(final PgConnection pgConnection) {
         return new ConfigurationMaintenanceOnHostImpl(pgConnection);
     }
+
```
Comment (mfvanek) [inline spring-boot-integration/pg-index-health-test-starter/src/main/java/io/github/mfvanek/pg/spring/DatabaseStructureHealthAutoConfiguration.java:275]: Redundant empty line

**Review by mfvanek — CHANGES_REQUESTED** [review]
Great job! It needs a little polish

**Comment by mfvanek** [comment]
Relates to https://github.com/mfvanek/pg-index-health/issues/317

File: spring-boot-integration/pg-index-health-test-starter/src/main/java/io/github/mfvanek/pg/spring/DatabaseStructureHealthAutoConfiguration.java (hunk 2 above)
Comment (mfaulther) [reply spring-boot-integration/pg-index-health-test-starter/src/main/java/io/github/mfvanek/pg/spring/DatabaseStructureHealthAutoConfiguration.java]: Fixed

**Review by mfvanek — APPROVED** [review]

**Pull request merged**